src/unity/lib/annotation/image_classification.cpp

```diff
@@ -95,11 +95,17 @@ annotate_spec::Annotations ImageClassification::getAnnotations(size_t start,
 
     label->imageclassificationlabel();
 
-    if (flex_data.at(i).get_type() == flex_type_enum::STRING) {
-      std::string label_value = flex_data.at(i).get<flex_string>();
+    const flexible_type& flex_label = flex_data.at(i);
+    if (flex_label.get_type() == flex_type_enum::UNDEFINED) {
+      // skip unlabeled items
+      continue;
+    }
+
+    if (flex_label.get_type() == flex_type_enum::STRING) {
+      std::string label_value = flex_label.get<flex_string>();
```

Comment:
Nit: can avoid a potential copy by using `const std::string&` here instead of `std::string`